web_export_view/static/src/js/web_export_view.js

```diff
@@ -85,7 +85,8 @@ openerp.web_export_view = function (instance) {
                                 }
                             }
                             else if (cell.classList.contains("oe_list_field_integer")) {
-                                export_row.push(parseInt(text));
+                                tmp2 = text.replace(instance.web._t.database.parameters.thousands_sep, "");
```

Comment:
Thanks for the clarification. I can agree with that, but I did check if this could then cause any clashes with other tmp2 variables in the core code. I did not find any, but I did find another example of thousands separator replacing, and it occurred to me that replace() in Javascript only replaces the first occurrence so this code is better:

https://github.com/odoo/odoo/blob/8.0/addons/web/static/src/js/formats.js#L245-L249

I suggest @bringsvor that you copy this code verbatim here to cover the case with more than one thousands separator.